### PR TITLE
TK248 - Added course and programs in dropdown using common function i…

### PR DESCRIPTION
**Pull request by karuna835**
…n add subscription, edit subscription and student report.

### CODE REVIEW

File: app/modules/course/bundle.php [hunk 1]
```diff
@@ -316,5 +316,98 @@ function course_bundle_get_by_category($category)
...
+			}
+
+			$get_student_count = $count;
+			$course["batch_student_count"] = $get_student_count;
+			$batches_code['course'][$course['course_id']][] = $course;
+		}
+	}
+	$bundle_array = ['programs' => $programs, 'batches_code' => $batches_code, 'courses' => $courses ];
+	return $bundle_array;
+	}
```
Comment (kailasb) [inline app/modules/course/bundle.php:410]: format code

File: app/modules/course/course.php [hunk 2]
```diff
@@ -42,24 +49,27 @@ function course_get_duration($combo, $combo_free = "", $bundle_id = "") {
...
+										else if (strcmp($res[0]['duration_unit'], 'months') == 0) $duration += intval($res[0]['duration_length']) * 30;
+										else if (strcmp($res[0]['duration_unit'], 'weeks') == 0) $duration += intval($res[0]['duration_length']) * 7;
+									}
 								}
 							}
-                    	}
+						}
                 	}
                 }
             }
```
Comment (kailasb) [inline app/modules/course/course.php:75]: Format code and remove extra line space

**Review by kailasb — CHANGES_REQUESTED** [review]
Check feedback & format code properly in all files

File: app/modules/course/bundle.php (hunk 1 above)
Comment (karuna835) [reply app/modules/course/bundle.php]: Code properly formatted

File: app/modules/course/course.php (hunk 2 above)
Comment (karuna835) [reply app/modules/course/course.php:75]: code properly formatted

**Review by kailasb — APPROVED** [review]
changes are proper

**Pull request merged**